### PR TITLE
Add clarification re: registering new Accept headers

### PR DESCRIPTION
**Pull request by ryangyoung**
Updates page to address comments in [this thread](https://github.com/payid-org/payid-documentation/pull/97#discussion_r458796702) about how/where to add new Accept headers.

### Context of Change

Adds more context/guidance on how to add new Accept headers.

### Type of Change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Refactor (non-breaking change that only restructures code)
- [ ] Tests (You added tests for code that already exists, or your new feature included in this PR)
- [x] Documentation Updates
- [ ] Release

### CODE REVIEW

**Comment by ryangyoung** [comment]
@Stormtv do you know what the expected check is that's not completing? Do I need to do something to trigger it?

**Comment by ryangyoung** [comment]
@Stormtv bumping this--not sure what's blocking the PR from merging. I can't see what checks are expected or not working. (Ticket to fix my gitlab access is [here](https://app.asana.com/0/1188384800557084/1188385166519462/f).)

**Comment by Stormtv** [comment]
> @Stormtv bumping this--not sure what's blocking the PR from merging. I can't see what checks are expected or not working. (Ticket to fix my gitlab access is [here](https://app.asana.com/0/1188384800557084/1188385166519462/f).)

@ryangyoung 
I broke something here when updating dependencies, i'll fix this right now

**Comment by Stormtv** [comment]
@ryangyoung Should be good now, make sure you `npm run lint` before creating the PR in the future.

File: docs/payid-headers.md [hunk 1]
```diff
@@ -29,7 +29,11 @@ Some different header options are shown here, with example values.
 | ACH      | `application/ach+json`          | { account: '363023456079',routing: '011302838'}                   |
 | All      | `application/payid+json`        | Variable, depending on the contents of each address               |
 
-If you create your own Accept header for another currency, follow the pattern for BTC Accept headers, and use three characters for the currency, as shown. In addition, consider creating a PR to update this list.
+If you create your own Accept header for another currency:
+
+1. Follow the pattern for BTC Accept headers and use three characters for the currency, as shown.
```
Comment (hbergren) [inline docs/payid-headers.md:34]: Nit: it's not strictly three characters, it's the "currency code" of the currency, so Chainlink would be LINK.

Comment (ryangyoung) [reply docs/payid-headers.md]: Good point. Updated.

**Pull request merged**